README.md

```diff
@@ -507,6 +513,12 @@ Manages configuration for Authorization Login Config Service.
 | N8k      | 7.0(3)F1(1)        | 1.3.0                  |
 | IOS XR   | unsupported        | unsupported            |
 
+#### <a name="cisco_aaa-caveats">Caveats</a>
+
+| Platform OS Caveat |
+|:-------------------|
+| Because of an OS issue, cisco_aaa* providers are not supported on N9K Platforms running OS Version 7.0(3)I3(1) |

```

Comment:
`cisco_aaa* providers are not currently supported on N9K Platforms running OS Version 7.0(3)I3(1) due to a known OS defect.`